kubric/core/base.py

```diff
@@ -76,16 +94,24 @@ def active_scene(self):
 
   @tl.default("uid")
   def _uid(self):
-    """ Use the class name and a global count separated by a period as the UID.
+    """Default initializer for uid trait.
+    
+    Use self.name and a global count separated by a period as the UID.
     Counting starts at 1 and always has at least three digits.
```

Comment:
Matching Blenders logic is fine here (and might actually be nicer, now that we support custom names), but the doc-string should reflect that behaviour.

Reply:
Q: When we match blender logic is this statement still true?
```lexicographical sorting of the first 999 instances```

Reply:
No, now it is the first 1000 instances :-)

Reply:
LoL, what I meant is whether the one without the suffix will be guaranteed to be the first, and I guess that's true.